mgmt_cli_test.py

```diff
@@ -188,25 +193,42 @@ def restore_backup(self, mgr_cluster, snapshot_tag, keyspace_and_table_list):  #
         else:
             raise ValueError(f'{backup_bucket_backend=} is not supported')
 
-        per_node_backup_file_paths = mgr_cluster.get_backup_files_dict(snapshot_tag)
-        for node in self.db_cluster.nodes:
+        nodetool_refresh_extra_flags = self.params.get('mgmt_nodetool_refresh_flags')
+        if not nodetool_refresh_extra_flags:
+            nodetool_refresh_extra_flags = ""
```

Comment:
can be reduced to `nodetool_refresh_extra_flags = self.params.get('mgmt_nodetool_refresh_flags') or ""`